_posts/2021-03-03-signalen-community-meetup-march-22.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Invitation: Signalen community meetup, March 22nd"
+date: 2021-03-03
+author: Alba Roza
+type: blogpost
+excerpt: New Signalen community day
+categories:
```

Comment:
Until we switch from categories to tags, each blogpost can only have 1 category.